eng/InstallRuntimes.proj

```diff
@@ -0,0 +1,109 @@
+<Project>
+  <!--
+     $(BuildArch) - architecture to test (x64, x86, arm, arm64). Defaults to x64.
+     $(DailyTest) - if true, only install/test the latest (master branch) runtime
+     
+     From Versions.props:
```

Comment:
This is sort of spooky action at a distance, now that it is not obvious why this doesn't import version but has access to all of this. This should probably become a targets file given that it's imported now.